gtd-timer/Services/ISignUpService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using gtdtimer.Timer.DAL.Entities;
+using gtdtimer.Timer.DTO;
+
+namespace gtdtimer.Services
+{
+    public interface ISignUpService : IBaseService
+    {
+        void Add(UserDTO model);
```

Comment:
please rename to 'AddUser' and 'GetUserById'

Reply:
Done